src/bindings/python/tests/test_onnx/test_ops_reshape.py

```diff
@@ -360,7 +357,7 @@ def test_split_1d():
 
     splits = np.array([2, 4]).astype(np.int64)
     node = onnx.helper.make_node(
-        "Split", inputs=["input", "splits"], outputs=["y", "z"], split=[2, 4]
+        "Split", inputs=["input", "splits"], outputs=["y", "z"]
```

Comment:
I believe at the time this code is written, onnx split is still in version-11 which has split_length as attribute rather than input, but in version-13, it becomes input. and specify this split as attributes will fails checks in `onnx.helper.make_node()`, it complains no attributes named "split", so I removed it.

https://github.com/onnx/onnx/blob/master/docs/Changelog.md#Split-11
https://github.com/onnx/onnx/blob/master/docs/Changelog.md#Split-13

Reply:
Thanks for explanation!